astropy_helpers/openmp_helpers.py

```diff
@@ -39,69 +41,222 @@
 }
 """
 
+def _get_flag_value_from_var(flag, var, delim=' '):
+    """
+    Utility to extract `flag` value from `os.environ[`var`]` or, if not present,
```

Comment:
use double backticks for parameters as a good practice, in public functions it would trip sphinx over otherwise (single backticks are for intersphinx links)

Reply:
Ok, cool, thanks for the intel, will do.